plugins/redis/env_var_and_flags_provisioner.go

```diff
@@ -0,0 +1,39 @@
+package redis
+
+import (
+	"context"
+
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+
+type EnvVarFlagsProvisioner struct {
+	sdk.Provisioner
+
+	Schema map[string]sdk.FieldName
+}
+
+func EnvVarFlags(schema map[string]sdk.FieldName) sdk.Provisioner {
```

Comment:
I'd prefer if this functionality existed in `sdk/provision/` and it would be shaped as:
1. A new util for flag provisioning similar to `EnvVars`. This can also take a `schema map[string]sdk.FieldName` as argument but the keys are the names of the flags.
2. A new util that chains a general number of provisioners together. This function can take a `[]Provisioner` and then it just loops through all of them and calls the `Provision` method.